micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/DigitJSONForm.js

```
@@ -255,110 +265,215 @@ const DigitJSONForm = ({
+      await secondFormatLocalizationMutation.mutateAsync({
+        params: {},
+        body: {
+          tenantId: tenantId,
+          messages: secondFormatMessages,
+        },
+      });
+    } catch (err) {
+      console.error("Second format localization upsert failed:", err);
+    }
+  }
+  onSubmit && onSubmit(transformedFormData, additionalProperties);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)